platform/W4.roc

```diff
@@ -173,7 +213,7 @@ text = \str, { x, y } ->
     |> Effect.map Ok
     |> InternalTask.fromEffect
 
-## Helper for colors when drawing text
+## Helper for setting the draw colors for text
```

Comment:
We probably should add a comment to all of the color related functions that they overwrite the existing DrawColors. As such, setting text color would change shape color, for example.

Reply:
Done